src/rooms/rooms.service.ts

```diff
@@ -0,0 +1,78 @@
+import { Injectable } from '@nestjs/common';
+import { CreateRoomDto } from './dto/create-room.dto';
+import { UpdateRoomDto } from './dto/update-room.dto';
+import { Room } from 'src/schemas/rooms.schema';
+import { InjectModel } from '@nestjs/mongoose';
+import mongoose, { FilterQuery, Model } from 'mongoose';
+import { ShowRoomDto } from './dto/show-room.dto';
+
+@Injectable()
+export class RoomsService {
+  constructor(@InjectModel(Room.name) private roomModel: Model<Room>) {}
+
+  async createRoom() {
+    const createdRoom = new this.roomModel({
+      title: 'tag12',
+      tagList: ['tag12', 'tag9', 'tag4'],
+      maxNum: 12,
+      currentNum: 10,
+      notice: 'awoeawi',
+      isPublic: true,
+      // password: '1234',
+      isChat: true,
+      imageUrl: '',
+      roomManager: new mongoose.Types.ObjectId('603d6f4e8b3f2a3d54f1b2e9'),
+      member: [new mongoose.Types.ObjectId('603d6f4e8b3f2a3d54f1b2e8')],
+    });
+    return createdRoom.save();
+  }
```

Comment:
```suggestion
async createRoom(createRoomDto: CreateRoomDto) {
  const createdRoom = new this.roomModel(createRoomDto);
  return createdRoom.save();
}
```

이 부분 하드코딩된 건 지워줘도 되지 않나요?